src/content/docs/installation/installation_t2macbook.mdx

```diff
@@ -3,98 +3,120 @@ title: T2 MacBook Installation
+:::
 
-### Getting WiFi firmware for the installation
+## Creating a macOS Recovery Installer
 
-<Steps>
-
-1. First boot into macOS and run this command:
-    ```sh
-    curl -sL https://wiki.t2linux.org/tools/firmware.sh | bash -s copy_to_efi
-    ```
-
-2. Now boot into the CachyOS ISO. Open Konsole there, and run these commands:
-    ```sh
-    sudo mkdir -p /tmp/apple-wifi-efi
-    sudo mount /dev/nvme0n1p1 /tmp/apple-wifi-efi
-    bash /tmp/apple-wifi-efi/firmware.sh get_from_efi
-    sudo umount /tmp/apple-wifi-efi
-    ```
-
-3. Setup `iwd` as backend for NetworkManager. You can run the following commands in Konsole to do so:
-    ```sh
-    cat <<EOF | sudo tee /etc/NetworkManager/conf.d/wifi_backend.conf
-    [device]
-    wifi.backend=iwd
-    EOF
-    sudo systemctl restart NetworkManager
-    ```
+It is highly recommended to create a bootable macOS installer **before** you modify your partitions or install CachyOS, especially if you are considering removing macOS entirely. This USB installer allows you to reinstall macOS for firmware updates, troubleshooting, or restoring your system if needed.
 
+<Steps>
+1.  **Obtain a USB Drive:** You'll need an external USB drive (at least 16GB, ideally 32GB).
+2.  **Follow Apple's Guide:** Apple provides detailed instructions on how to download a macOS installer and create a bootable USB drive from it.
+    *   **Follow the official guide here:** [How to create a bootable installer for macOS - Apple Support](https://support.apple.com/en-us/101578)
+3.  **Store Safely:** Keep this USB drive in a safe place in case you need to restore macOS later.
 </Steps>
 
-Now you can connect to a WiFi network and proceed with the installation.
+## Pre-Installation
 
-### Proceeding with Installation
+Perform these steps within macOS **before** booting the CachyOS installer. Ensure you have created your [macOS Recovery Installer](#creating-a-macos-recovery-installer) if desired.
 
-Follow the instructions in [Installation Prepare](/installation/installation_prepare#creating-a-bootable-cachyos-usb-drive) on how to download
-the ISO and create a bootable USB drive.
-
-You need to follow additional steps to prepare a T2 MacBook for installing Linux such as partitioning your disk using macOS and disabling secure boot. You can follow the [t2linux Preinstall guide](https://wiki.t2linux.org/guides/preinstall/) for instructions.
+<Steps>
 
-Refer to [Installation on Root](/installation/installation_on_root) after creating a bootable USB drive.
-CachyOS applies necessary boot parameters and configurations to the user's T2 MacBook with [CachyOS Hardware Detection](/features/chwd).
+1.  **Prepare Your Disk:**
+    *   If you plan to dual-boot (recommended), use macOS Disk Utility to resize your existing macOS partition and create free space for CachyOS.
+        * Refer to the [t2linux Partition with Disk Utility guide](https://wiki.t2linux.org/guides/preinstall/#partition-with-disk-utility) for detailed instructions.
```

Comment:
If you are doing a overhaul, the partition steps should be in this guide rather than putting a link